### ISSUE TITLE
Not all game scenes have been tested

### ISSUE THREAD

**Issue by JayFoxRox** [JayFoxRox/Lindbergh-Emulator#90]
This should be ran for at least a week in a good environment by a person who is very familar with the game. Screenshots should be taken.

**Comment by HeroponRikiBestest** [comment]
@JayFoxRox Sorry to bother you, would you still want testing? I'm a little unclear on what you'd actually want tested, but I'd be happy to help with that.

**Comment by JayFoxRox** [comment]
No. This project is dead (until someone actually takes it over, which is unlikely to happen).

The issues were carried over from my internal repo for when this was going to be sold to someone (which was meant to support OR2 and LGJ).
Because nobody is paying for this and because there's no development on it, there's no point in testing / preparing for a final version (which would require QA, which this issue was meant to be a part of).
